### PR TITLE
Updated changelogs to reflect STG 80 features

### PR DESCRIPTION
**Pull request by seanmcc-msft**

### CODE REVIEW

**Review by jaschrep-msft — APPROVED** [review]

File: sdk/storage/Azure.Storage.Files.DataLake/CHANGELOG.md [hunk 1]
```diff
@@ -1,14 +1,8 @@
...
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- Added supprot for service version 2021-02-12.
+- Added support for listing system file systems with DataLakeServiceClient.GetFileSystmes() and .GetFileSystemsAsync().
```
Comment (amnguye) [inline sdk/storage/Azure.Storage.Files.DataLake/CHANGELOG.md:5]: DataLakeServiceClient.GetFileSystmes() should be DataLakeServiceClient.GetFileSystems()

**Review by amnguye — APPROVED** [review]

File: sdk/storage/Azure.Storage.Files.DataLake/CHANGELOG.md (hunk 1 above)
Comment (seanmcc-msft) [reply sdk/storage/Azure.Storage.Files.DataLake/CHANGELOG.md]: Fixed.

**Pull request merged**